Eastern-Research-Group/mywaterway
# Feature/463 update cypress tests

## Related Issues:
* [HMW-463](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-463)

## Main Changes:
* Updated previous tests that were failing due to new changes, most of which were related to duplicate text with the layer names in the Save panel of the Add/Save Data widget.
  * One bug was related to state management on the Monitoring Report page. That issue was resolved, too.
* Added a visual regression test for the Surrounding Features widget.
* Added a test for the `WaterbodyListVirtualized` component on the state Advanced Search page.
* Added a test for the watershed plans table on the _Restore_ tab.
* Added a test for the dischargers toggle table on the _Overview_ tab.
* Added tests for the list of switches in the Save panel of the Add/Save Data widget.

## Steps To Test:
1. Review the tests, run them, and verify they pass.
2. Verify that the Monitoring Report page and the `AccordionItem` components still function as expected, since slight changes were made to both.
